src/utils/sliceMultiLangData.js

```diff
@@ -0,0 +1,11 @@
+export default function sliceMultiLangData(title, lang) {
+  const searchTerm = `[:${lang}]`;
+  const langIndex = title.indexOf(searchTerm);
+  if (langIndex !== -1) {
+    const startIndex = langIndex + searchTerm.length;
+    const endIndex = title.indexOf('[:', startIndex);
+
+    return title.slice(startIndex, endIndex);
+  }
+  return title;
```

Comment:
title should be renamed to value since it could be anything  just to keep the naming conventions